### PR TITLE
Prl 603 screen refinement part1

### PR DESCRIPTION
**Pull request by CatherineChowWingMei**
**Before creating a pull request make sure that:**

- [X] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [X] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###

PRL-603 Screen refinement Part 1  

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: charts/prl-cos/values.preview.template.yaml [hunk 1]
```diff
@@ -21,7 +21,7 @@ java:
     IDAM_CLIENT_REDIRECT_URI: https://prl-cos-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal/oauth2/callback
     IDAM_API_URL: https://idam-api.{{ .Values.global.environment }}.platform.hmcts.net
     IDAM_S2S_AUTH_URL: http://rpe-service-auth-provider-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
-    CORE_CASE_DATA_API_URL: http://ccd-data-store-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
+    CORE_CASE_DATA_API_URL: http://ccd-data-store-api-prl-ccd-definitions-pr-100.service.core-compute-preview.internal
```
Comment (yogendraU) [inline charts/prl-cos/values.preview.template.yaml]: Hi @CatherineChowWingMei , can you please revert this change to what is in master.

**Review by yogendraU — APPROVED** [review]

**Review by AlokDatta — APPROVED** [review]
